src/js/api/config.js

```diff
@@ -61,9 +61,50 @@ function _adjustDefaultBwEstimate(estimate) {
     return Defaults.bandwidthEstimate;
 }
 
+function _copyToLocalization(allOptions) {
+    const { advertising, related, sharing } = allOptions;
+    const localization = Object.assign({}, Defaults.localization, allOptions.localization);
+
+    if (advertising) {
+        localization.advertising = Object.assign({}, localization.advertising, {
+            admessage: advertising.admessage,
+            cuetext: advertising.cuetext,
+            loadingAd: localization.loadingAd,
+            podmessage: advertising.podmessage,
+            skipmessage: advertising.skipmessage,
+            skiptext: advertising.skiptext
+        });
+    }
+
+    if (related) {
+        if (typeof localization.related === 'object') {
+            localization.related = Object.assign({}, localization.related, {
+                autoplaymessage: related.autoplaymessage,
+                heading: localization.nextUp || related.heading,
```

Comment:
`related.heading` does not appear in the documentation; i say we don't need to backwards support it

Reply:
why would we use `localization.nextUp`?